kickstarts/partials/post/python_modules.ks.erb

```diff
@@ -45,84 +39,30 @@ azure-mgmt-sql==0.10.0
 azure-mgmt-storage==3.1.0
 azure-mgmt-trafficmanager==0.50.0
 azure-mgmt-web==0.41.0
-azure-nspkg==2.0.0
 azure-storage==0.35.1
 backports.ssl-match-hostname==3.5.0.1
-bcrypt==3.1.4
 boto==2.47.0
 boto3==1.6.2
-botocore==1.9.3
-cachetools==3.0.0
-certifi==2018.1.18
-cffi==1.14.6
-chardet==3.0.4
-colorama==0.3.9
-cryptography==2.8 # Match the system python38-cryptography version
-decorator==4.2.1
 deprecation==2.0
-docutils==0.14
-dogpile.cache==0.6.5
 google-auth==1.6.2
-humanfriendly==4.8
-idna==2.6
 ipaddress==1.0.19
-iso8601==0.1.12
-isodate==0.6.0
-Jinja2==2.11.3 # Match the system python38-jinja2 version
-jmespath==0.9.3
-jsonpatch==1.21
-jsonpointer==2.0
-keystoneauth1==3.11.2
-knack==0.3.3
-lxml==4.9.1
-MarkupSafe==1.1.1
 monotonic==1.4
-msrest==0.6.1
-msrestazure==0.5.0
-munch==2.2.0
 ncclient==0.6.3
 netaddr==0.7.19
-netifaces==0.10.6
-ntlm-auth==1.0.6
-oauthlib==3.2.0
 openstacksdk==0.23.0
-os-service-types==1.2.0
 ovirt-engine-sdk-python==4.2.4
-packaging==17.1
-paramiko==2.10.1
-pbr==3.1.1
 pexpect==4.6.0
 psutil==5.6.6 # Match the version installed directly into the venv
-ptyprocess==0.5.2
-pyasn1==0.4.2
-pyasn1-modules==0.2.3
-pycparser==2.18
-Pygments==2.7.4
-PyJWT==2.4.0
 pykerberos==1.2.1
-PyNaCl==1.2.1
-pyOpenSSL==17.5.0
-pyparsing==2.2.0
-python-dateutil==2.6.1
 pyvmomi==6.5
 pywinrm==0.3.0
-PyYAML==5.4.1 # Match the system python38-pyyaml version
 requests==2.25.1
 requests-credssp==0.1.0
 requests-kerberos==0.14.0
-requests-ntlm==1.1.0
-requests-oauthlib==0.8.0
-requestsexceptions==1.4.0
-s3transfer==0.1.13
-selectors2==2.0.1
-six==1.11.0
-stevedore==1.28.0
-tabulate==0.7.7
-urllib3==1.26.5
-xmltodict==0.11.0
 EOF
 
   pip3.8 install -r requirements.txt
+  pip3.8 list --format freeze > installed.txt
```

Comment:
Should this have a more specific name and get dropped in the manifest directory?

Reply:
Manifest was definitely my motivation for this.  For now though it's a simple file that just lives next to the requirements.txt.

I plan to do some manifest stuff in a separate PR.